utils/db_setup/isCourseInstance.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE "isCourseInstanceOf"(
+       id       serial,
+       name     varchar(120) REFERENCES "Course"(name),
```

Comment:
is there a reason this is name and not something like `course_name` to match the `ci_` names? Is it just because of the name between the two `name` columns?

Reply:
Yep, I assumed when thinking of the relation `isCourseInstanceOf` I was thinking that the `name` would be synonymous with the Course's name. We can change that to `course_name` if it makes it more explicit for us.

Reply:
100% in favor of more explicit